$NetBSD: patch-ad,v 1.2 2005/03/16 20:17:50 rillig Exp $

Probably the (...) meant: I don't use the arguments. But c++-2.95.3
complained about the wrong function type.

--- xkobo.C.orig	Fri Jun 14 19:27:27 1996
+++ xkobo.C	Wed Mar 16 20:58:07 2005
@@ -51,7 +51,7 @@ int             cheat_mode;
 
 static int      signal_delivered = 1;
 
-static void sig_handle(...)
+static void sig_handle(int)
 {
     signal_delivered = 1;
 }
